test/unit/status.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import {describe, it} from 'mocha';
+import {rpcCodeFromHttpStatusCode} from '../../src/status';
+
+describe('status.ts', () => {
+  it('rpcCodeFromHttpStatusCode', () => {
+    assert.deepStrictEqual(rpcCodeFromHttpStatusCode(200), 0);
```

Comment:
nit: if possible, can we use gRPC constants here in this test?  Otherwise, it reads a little bit cryptic :) Other than that, LGTM, thank you!